module/VuFind/src/VuFindTest/Feature/ConfigRelatedServicesTrait.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+/**
+ * Trait for tests involving config related services.
+ *
+ * PHP version 8
+ *
+ * Copyright (C) The National Library of Finland 2022.
```

Comment:
Should we update the date here since this is a new file?

```suggestion
 * Copyright (C) The National Library of Finland 2022-2025.
```